libvast/src/command.cpp

```diff
@@ -351,24 +351,30 @@ parse(const command& root, command::argument_iterator first,
   return result;
 }
 
+/// Merge settings of `src` into `dst`, overwriting existing values
+/// from `dst` if necessary.
+static void
```

Comment:
Are we doing `static` over unnamed namespaces officially now?

Reply:
Apparently not ;)